Sorting/Bucket Sort/bucket sort.cpp

```diff
@@ -0,0 +1,49 @@
+#include<bits/stdc++.h>
+using namespace std;
+void temp_sort(vector<int>&arr,int n,int val)
+{
+    vector<vector<int>>buckets(11);
```

Comment:
Can the number of buckets be a variable?

Reply:
no it will always be equal to 10 as the single digits can be from 0 to 9 so more or less we can assign 10 to a variable and then place that variable. I have wrote 11 by mistake but still it will work fine and correct (just a free block will be created)